### PR TITLE
Make Express Script Reusable

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
We already ship our scripts in terra-dev-site. Update such that other repos can use it as well and we can remove their express scripts.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -71,6 +71,7 @@
     "clean-webpack-plugin": "^0.1.17",
     "commander": "^2.9.0",
     "css-loader": "^0.28.7",
+    "express": "^4.16.3",
```
Comment (mjhenkes) [inline package.json]: We didn't have express already added as a dependency? that seems like a miss.

Comment (emilyrohrbough) [reply package.json]: Yeah?? I think since we were only using it for heroku at this point it handled it for us?

File: scripts/express/app.js [hunk 2]
```diff
@@ -1,12 +1,11 @@
-/* eslint-disable import/no-extraneous-dependencies */
 /* eslint-disable no-console */
 const express = require('express');
 const path = require('path');
 
 const app = express();
 const port = process.env.PORT || 8081;
 // path to webpack built path
-const buildPath = path.join(__dirname, '..', '..', 'build');
+const buildPath = path.join(process.cwd(), '..', '..', 'build');
```
Comment (mjhenkes) [inline scripts/express/app.js]: Actually, i don't think you want to go down from the process.cwd() dir. Probably just 
`path.join(process.cwd(), 'build')`

**Comment by emilyrohrbough** [comment]
Going to test in clincial and see if herkou kicks off successfully

**Comment by emilyrohrbough** [comment]
verified! see https://github.com/cerner/terra-clinical/pull/317

**Pull request merged**